packages/common/src/authorization/organizationMemberAbility.ts

```diff
@@ -216,24 +388,6 @@ const applyOrganizationMemberStaticAbilities: Record<
         can('manage', 'Validation', {
             organizationUuid: member.organizationUuid,
         });
-        can('promote', 'SavedChart', {
```

Comment:
These are useless because users already have the ability to `manage` (see line 280 in the right-side of the diff) and manage means that any action is allowed (in other words, if you can manage you can always promote). So this was adding a redundant rule